Incompatible change in the com.sun.ts.tests.signaturetest.javaee.JavaEESigTest class

**Describe the bug**
The changes in #1209 to change the SigTestEE#setup(Sring[], Properties) method to setup() has broken the ServiceEETest/EETest invocation protocol and does not allow the existing vehicles to be used.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Expected behavior**
The EE10 vehicle test method dispatch mechanism needs to stay in place to work with the javatest protocol used in the porting tools


**Additional context**
Running the com.sun.ts.tests.signaturetest.javaee.JavaEESigTest in an appclient vehicle fails with:

```bash
19:33:10,941 INFO  [org.jboss.as.server] (Thread-45) WFLYSRV0010: Deployed "JavaEESigTest_appclient_vehicle.ear" (runtime-name : "JavaEESigTest_appclient_vehicle.ear")
19:33:10,945 INFO  [stdout] (Thread-188) ************************************************************
19:33:10,946 INFO  [stdout] (Thread-188) * props file set to "/tmp/starksm-cts-props.txt"
19:33:10,946 INFO  [stdout] (Thread-188) ************************************************************
19:33:10,948 INFO  [stdout] (Thread-188) 07-14-2024 19:33:10:  ERROR: Could not find setup method fortestcase: signatureTest
19:33:10,948 INFO  [stdout] (Thread-188) 07-14-2024 19:33:10:  ERROR: Exception at:  
19:33:10,948 INFO  [stdout] (Thread-188) 07-14-2024 19:33:10:  ERROR: java.lang.NoSuchMethodException: com.sun.ts.tests.signaturetest.javaee.JavaEESigTest.setup([Ljava.lang.String;,java.util.Properties)
19:33:10,948 INFO  [stdout] (Thread-188) 	at java.base/java.lang.Class.getMethod(Class.java:2395)
19:33:10,948 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.getSetupMethod(EETest.java:146)
19:33:10,948 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.run(EETest.java:547)
19:33:10,948 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.ServiceEETest.run(ServiceEETest.java:115)
19:33:10,948 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.getPropsReady(EETest.java:486)
19:33:10,949 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.ServiceEETest.run(ServiceEETest.java:209)
19:33:10,949 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.run(EETest.java:285)
19:33:10,949 INFO  [stdout] (Thread-188) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.javaee.JavaEESigTest.main(JavaEESigTest.java:532)
19:33:10,949 INFO  [stdout] (Thread-188) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
19:33:10,949 INFO  [stdout] (Thread-188) 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
19:33:10,949 INFO  [stdout] (Thread-188) 	at org.jboss.as.appclient@33.0.0.Beta1-SNAPSHOT//org.jboss.as.appclient.service.ApplicationClientStartService$1.run(ApplicationClientStartService.java:89)
19:33:10,949 INFO  [stdout] (Thread-188) 	at java.base/java.lang.Thread.run(Thread.java:1583)
```

With the changes in the #1367 PR, the sigtest appclient vehicle fails with:

```bash
20:10:32,819 INFO  [stdout] (Thread-150) 07-14-2024 20:10:32:  ********** VALIDATE IN STATIC MODE - TO CHECK CONSANT VALUES ****
20:10:32,819 INFO  [stdout] (Thread-150) 07-14-2024 20:10:32:  Static mode supports checks of static constants values 
20:10:32,819 INFO  [stdout] (Thread-150) 07-14-2024 20:10:32:  ERROR: Unexpected exception /sig-test.map (No such file or directory)
20:10:32,819 INFO  [stdout] (Thread-150) 07-14-2024 20:10:32:  ERROR: Test case throws exception: signatureTest failed with an unexpected exception
20:10:32,819 INFO  [stdout] (Thread-150) 07-14-2024 20:10:32:  ERROR: Exception at:  
20:10:32,820 INFO  [stdout] (Thread-150) 07-14-2024 20:10:32:  ERROR: java.lang.Exception: signatureTest failed with an unexpected exception
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.SigTestEE.signatureTest(SigTestEE.java:333)
20:10:32,820 INFO  [stdout] (Thread-150) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
20:10:32,820 INFO  [stdout] (Thread-150) 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.run(EETest.java:596)
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.ServiceEETest.run(ServiceEETest.java:115)
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.getPropsReady(EETest.java:486)
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.ServiceEETest.run(ServiceEETest.java:209)
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.lib.harness.EETest.run(EETest.java:285)
20:10:32,820 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.javaee.JavaEESigTest.main(JavaEESigTest.java:532)
20:10:32,820 INFO  [stdout] (Thread-150) 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
20:10:32,820 INFO  [stdout] (Thread-150) 	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
20:10:32,820 INFO  [stdout] (Thread-150) 	at org.jboss.as.appclient@33.0.0.Beta1-SNAPSHOT//org.jboss.as.appclient.service.ApplicationClientStartService$1.run(ApplicationClientStartService.java:89)
20:10:32,820 INFO  [stdout] (Thread-150) 	at java.base/java.lang.Thread.run(Thread.java:1583)
20:10:32,821 INFO  [stdout] (Thread-150) Caused by: java.io.FileNotFoundException: /sig-test.map (No such file or directory)
20:10:32,821 INFO  [stdout] (Thread-150) 	at java.base/java.io.FileInputStream.open0(Native Method)
20:10:32,821 INFO  [stdout] (Thread-150) 	at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
20:10:32,821 INFO  [stdout] (Thread-150) 	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
20:10:32,821 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.SignatureTestDriver.loadMapFile(SignatureTestDriver.java:460)
20:10:32,821 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.SignatureTestDriver.getSigFileInfo(SignatureTestDriver.java:549)
20:10:32,821 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.SigTestDriver.createTestArguments(SigTestDriver.java:104)
20:10:32,821 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.SignatureTestDriver.executeSigTest(SignatureTestDriver.java:192)
20:10:32,821 INFO  [stdout] (Thread-150) 	at deployment.JavaEESigTest_appclient_vehicle.ear.JavaEESigTest_appclient_vehicle_client.jar//com.sun.ts.tests.signaturetest.SigTestEE.signatureTest(SigTestEE.java:285)
20:10:32,821 INFO  [stdout] (Thread-150) 	... 12 more
```

I don't know if there are other missing changes or if the way in which the sig map files are made available needs to be updated.

--- a/common/pom.xml
+++ b/common/pom.xml
@@ -98,6 +98,7 @@
                     <include>**/*.properties</include>
                     <include>**/*.jsp</include>
                     <include>**/client.html</include>
+                    <include>**/*.xhtml</include>
                 </includes>
                 <excludes>
                     <exclude>**/build.xml</exclude>

